CHANGELOG.md

```diff
@@ -23,6 +23,9 @@ Main (unreleased)
 - `otelcol.exporter.prometheus`: Set `include_scope_info` to `false` by default. You can set 
   it to `true` to preserve previous behavior. (@gouthamve)
 
+- `prometheus.remote_write`: Set `retry_on_http_429` to `true` by default in the `queue_config` block.
+  You can set it to `false` to preserve previous behavior. (@wildum)
```

Comment:
I think we always add migration instructions to the upgrade guide as described above. Could you add some there? I think that we should just say that no action is required if you were setting the `retry_on_http_429` explicitly. If you did not set it explicitly, but you would like to _not_ retry on HTTP 429, you should set it to false as you upgrade to this version.